lib/sparkql/token.rb

```diff
@@ -10,7 +10,8 @@ module Sparkql::Token
   DECIMAL = /^\-?[0-9]+\.[0-9]+/
   CHARACTER = /^'([^'\\]*(\\.[^'\\]*)*)'/
   DATE = /^[0-9]{4}\-[0-9]{2}\-[0-9]{2}/
-  DATETIME = /^[0-9]{4}\-[0-9]{2}\-[0-9]{2}T[0-9]{2}\:[0-9]{2}(\:[0-9]{2})?(\.[0-9]{1,50})?(((\+|-)[0-9]{2}\:?[0-9]{2})|Z)?/
+  TIME = /^[0-9]{2}\:[0-9]{2}((\:[0-9]{2})(\.[0-9]{1,50})?)?/
```

Comment:
Nixing the timezone is intentional here and matches up with `timeOfDayValue` type specified in odata. Since we're not tracking days, things get a little weird when you compare zones that wrap days (e.g. 16:00:00-06 Gt 16:00:00+09). Also if you play with the `timetz` type in postgres, things don't seem to match correctly.

We could maybe add in some sort of support if it deemed to be needed for something specific, but I plan to stick with what's needed (and works!) for now :-D